zanata-war/src/main/webapp/WEB-INF/layout/admin/contact_admin_modal.xhtml

```diff
@@ -7,78 +7,62 @@
 
   <script>
-    function cancelEmail() {
-      document.getElementById('#{rich:clientId('subject')}').value = '';
-      document.getElementById('#{rich:clientId('message')}').value = '';
+    $(document).ready(function() {
+      jQuery('#' + jqSelector('#{rich:clientId('contact-admin-subject')}')).
+        val('#{msgs['jsf.message.admin.inquiry.subject']}');
+    });
 
-      jQuery('#{rich:clientId('send')}').removeClass('is-active');
+    function cancelContactAdmin() {
+      jQuery('#' + jqSelector('#{rich:clientId('contact-admin-message')}')).val('');
+
+      jQuery(getContactAdminForm()).find('#send-button').removeClass('is-active');
       return false;
     }
+
+    function getContactAdminForm() {
+      return jQuery('#' + jqSelector('#{rich:clientId('contactAdminForm')}'));
+    }
   </script>
 
   <div class="modal__dialog l--constrain-medium">
     <header class="modal__header">
       <h2 class="modal__title">
-        #{msgs['jsf.ContactAdmin']}
+        #{msgs['jsf.message.admin.inquiry.title']}
       </h2>
       <button type="button"
         class="modal__close button--link"
-        data-dismiss="modal" onclick="cancelEmail();return false;">
+        data-dismiss="modal" onclick="cancelContactAdmin();return false;">
         <i class="i i--huge i--cancel"></i>
       </button>
     </header>
 
     <h:form id="contactAdminForm" styleClass="l--push-bottom-0">
-      <div class="modal__content l--pad-all-quarter">
-        <div class="l--push-top-half form__item">
-          <div class="txt--understated">
-            #{msgs['jsf.email.ReplyAddress']} <span class="txt--meta  l--push-right-quarter">
-          #{msgs['jsf.email.ReplyAddress.description']}</span>
-            #{contactAdminAction.replyEmail}
-          </div>
-        </div>
-
-        <div class="l--push-top-half form__item">
-          <h:outputLabel for="subject">
-            #{msgs['jsf.email.Subject']}
-          </h:outputLabel>
-          <s:decorate id="subjectField" template="../field.xhtml">
-            <h:inputText id="subject" value="#{contactAdminAction.subject}"
-              required="true">
-              <a4j:ajax event="blur" render="subjectField" execute="@this"/>
-            </h:inputText>
-          </s:decorate>
-        </div>
-
-
-        <div class="l--push-top-half form__item">
-          <h:outputLabel for="message">
-            #{msgs['jsf.email.MessageBody']}
-          </h:outputLabel>
-          <s:decorate id="messageField" template="../field.xhtml">
-            <h:inputTextarea id="message" value="#{contactAdminAction.htmlMessage}"
-              required="true">
-              <a4j:ajax event="blur" render="messageField" execute="@this"/>
-            </h:inputTextarea>
-          </s:decorate>
-        </div>
+      <h:inputHidden id="contact-admin-subject" value="#{contactAdminAction.subject}"/>
+      <div class="modal__content l--pad-all-1">
+        <s:decorate id="messageField" template="../field.xhtml">
```

Comment:
We should not use relative paths to the generic templates like this. All the template paths should be relative to the `zanata-war/src/main/webapp` directory except when the template is specific to a particular component, so that they don't break if we change the directory structure of some xhtml pages.

I think this should be `/WEB-INF/layout/field.xhtml`